aep/general/0002/aep.md

```diff
@@ -0,0 +1,39 @@
+# AEP Numbering
+
+This document describes the numbering system.
+
+AEP provides a numbering system that:
+
+- Groups AEPs primarily by their purpose, and secondarily by the general theme
+  they are discussing.
+- Allows for new AEPs to be added incrementally, without adjusting the numbers
+  of existing AEPs.
+
+## AEP Blocks
+
+The list of blocks are:
+
+| Range     | Name            | Description                                                                                                                                                                           |
+| --------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1-999     | meta topics     | topics that are not directly related to API guidance, but provide guidance or clarity for the AEPs themselves                                                                         |
+| 1000-1999 | specification   | APIs which are AEP-compliant **must** adhere to these AEPs. Clients authors may read these AEPs to determine what behavior they can rely on when integrating with AEP-compliant APIs. |
```

Comment:
I thought we had tentatively concluded that this was duplicative of **must** vs. **should**, and so maybe we weren't going to do it?

Reply:
Hmm... wires crossed? I definitely don't see the distinction as redundant: we should point at which AEPs one should strictly adhere to.

I don't think this would work otherwise: e.g. if we decided on singletons being a design pattern, are you okay with every single clause in that AEP being a **should**?

Reply:
Discussed in weekly meeting, but for posterity:  we discussed this live and decided to go with a different model (something more like AEP states, plus drafts of new AEPs being able to replace existing AEPs).

Marking thread resolved since it's obsolete.